src/middleware/is.psc.validation.middleware.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from "express";
+import { URL_QUERY_PARAM } from "../types/page.urls";
+import { isPscFlagValid } from "../validators/is.psc.validator";
+import { logger } from "../utils/logger";
+import { Templates } from "../types/template.paths";
+
+export const isPscQueryParameterValidationMiddleware = (req: Request, res: Response, next: NextFunction) => {
+  logger.debug("Execute isPsc query parameter validation middleware checks");
+
+  const isPsc: string = req.query[URL_QUERY_PARAM.IS_PSC] as string;
+
+  if (!isPsc) {
+    return next();
+  }
+
+  logger.debug("Check isPsc");
+  if (!isPscFlagValid(isPsc)) {
+    logger.errorRequest(req, "No valid isPsc query parameter supplied: " + req.originalUrl);
```

Comment:
@markpit I think I'll need to make use of your new common function to truncate the request content at this point, once merged.

Reply:
This now added and will be present in next commit.